core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -29,6 +29,18 @@ import org.apache.kafka.common.internals.Topic
 
 class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareTest {
 
+  @Before
+  override def setUp() {
+    super.setUp()
+    zkClient.createControllerEpochRaw(1)
```

Comment:
Do we need these changes?. tests are passing without this changes.

Reply:
Removed.